packages/create-sitecore-jss/src/templates/nextjs-styleguide/src/components/fields/Styleguide-FieldUsage-Image.tsx

```diff
@@ -39,18 +40,24 @@ const StyleguideFieldUsageImage = (props: StyleguideFieldUsageImageProps): JSX.E
     />
 
     {/*
-      Srcset adaptive image usage example
-      Adaptive srcsets are supported using Sitecore server-side resizing.
-      The `srcSet` can use Sitecore image resizing parameters (i.e. w, h, mw, mh).
-      Sample create a srcset using two sizes (server resizing), 300 and 100px max widths, respecting aspect ratio.
-      IMPORTANT: srcSet params must be whitelisted for adaptive resizing to occur. See /sitecore/config/*.config (search for 'allowedMediaParams')
+      srcSet in Nextjs Image is set inside of the next.config by setting an array of deviceSizes inside the images option. See here: https://nextjs.org/docs/api-reference/next/image#device-sizes
+      These sizes should match your Sitecore server-side allowlist.
+      See /sitecore/config/*.config (search for 'allowedMediaParams')
+       image sizes are set similarly: https://nextjs.org/docs/api-reference/next/image#image-sizes
```

Comment:
Seems like this (imageSizes) should be mentioned alongside deviceSizes in note above.

Reply:
It will use the default imageSizes if we don't pass anything.
We only need to pass it we need to override it with the default values.